### PR TITLE
Add VIP-Go standards to `phpcs.xml.dist`

### PR DESCRIPTION
**Pull request by kienstra**
* Add [WordPress-VIP-Go](https://github.com/Automattic/VIP-Coding-Standards/) standards
* Fix an issue that those standards raised in `site-counts.php`

### CODE REVIEW

File: site-counts.php [hunk 1]
```diff
@@ -16,5 +16,4 @@
 	require_once __DIR__ . '/vendor/autoload.php';
 }
 
-$plugin = new Plugin( __FILE__ );
-add_action( 'plugins_loaded', [ $plugin, 'init' ] );
+add_action( 'plugins_loaded', [ new Plugin( __FILE__ ), 'init' ] );
```
Comment (kienstra) [inline site-counts.php:19]: The `WordPress-VIP-Go` standard raised this error:

```
Overriding WordPress globals is prohibited. Found
   |       | assignment to $plugin
   |       | (WordPress.WP.GlobalVariablesOverride.Prohibited)
```

Comment (kienstra) [reply site-counts.php:19]: It seems like a false positive, as `$plugin` isn't a global. 

But it's just as easy to remove the variable entirely.

File: composer.lock [hunk 2]
```diff
@@ -4,20 +4,20 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "3bdddda79f9b227b4522cc36b1ea0fdf",
+    "content-hash": "35cc5c104af14e1568d400168d6f4d4e",
```
Comment (kienstra) [inline composer.lock:7]: This diff is more extensive than I'd like.

It's from `composer update` after requiring `automattic/vipwpc`.

It looks like it updated several other pacakges.

File: site-counts.php (hunk 1 above)
Comment (kasparsd) [reply site-counts.php:19]: We could name it something like ‘$xwp_challange_plugin’ which would be unique enough. The benefit of having a variable is that it allows additional bootstrapping and class configuration before the instance is used for filters, etc. It isn’t useful now but could be in the future.

Comment (kienstra) [reply site-counts.php:19]: Thanks, that's a good idea to have `$xwp_challenge_plugin` for more bootstrapping.

**Pull request merged**